src/os/CMakeLists.txt

```diff
@@ -138,7 +138,8 @@ if(WITH_PMEM)
   add_library(pmem STATIC IMPORTED GLOBAL)
   add_dependencies(pmem nvml_ext)
   set_target_properties(pmem PROPERTIES
-    IMPORTED_LOCATION "${CMAKE_BINARY_DIR}/src/nvml/src/nondebug/libpmem.a")
+    IMPORTED_LOCATION "${CMAKE_BINARY_DIR}/src/nvml/src/nondebug/libpmem.a"
+    IMPORTED_LINK_INTERFACE_LIBRARIES ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
`IMPORTED_LINK_INTERFACE_LIBRARIES` is deprecated, please consider using `INTERFACE_LINK_LIBRARIES` instead. see https://cmake.org/cmake/help/v2.8.12/cmake.html#prop_tgt:IMPORTED_LINK_INTERFACE_LIBRARIES .